apps/web/src/hooks/affine/use-affine-helper.ts

```diff
@@ -0,0 +1,70 @@
+import { useCallback } from 'react';
+
+import type { BlockSuiteWorkspace } from '../../shared';
+import { usePageMeta, usePageMetaHelper } from '../use-page-meta';
+
+export function useAffineHelper(
```

Comment:
Why we have a `Affine` helper but doesn't use any API of affine. I think this should be moved to normal `metaHelper`